dev/ScrollBar/ScrollBar.vcxitems

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <HasSharedItems>true</HasSharedItems>
+    <ItemsProjectGuid>{8E0ACCD0-B881-488F-B449-0C4F89B55857}</ItemsProjectGuid>
+  </PropertyGroup>
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);$(MSBuildThisFileDirectory)</AdditionalIncludeDirectories>
+    </ClCompile>
+  </ItemDefinitionGroup>
+  <ItemGroup>
+    <ProjectCapability Include="SourceItemsFromImports" />
+  </ItemGroup>
+  <ItemGroup Condition="$(BuildLeanMuxForTheStoreApp) != 'true'">
+    <Page Include="$(MSBuildThisFileDirectory)ScrollBar_rs1_themeresources.xaml">
+      <Version>RS1</Version>
+      <Type>ThemeResources</Type>
+    </Page>
+    <Page Include="$(MSBuildThisFileDirectory)ScrollBar_rs2_themeresources.xaml">
+      <Version>RS2</Version>
+      <Type>ThemeResources</Type>
+    </Page>
+    <Page Include="$(MSBuildThisFileDirectory)ScrollBar_rs3_themeresources.xaml">
+      <Version>RS3</Version>
```

Comment:
If it's possible, copy the latest style for scrollbar and then apply it down-level to rs1 by conditional xaml.
We don't need to maintain multiple version of scrollbar.

Reply:
I'll try coping the latest template and see if anything breaks...
Do we have script to merge ThemeDictionaries?

Reply:
> Can we add test coverage for these changes? Even if the test is just to navigate to a Page the contains these controls to ensure that the template can be applied without crashing. Or do we already have such a test?

Added a simple test to just open the CornerRadiusPage and verify it doesn't crash. I'll add tests to verify the actual corner radius values once we have the numbers in a centralized theme rousource.